### PR TITLE
fixes simultaneous playing of tabs when unpause intent is given

### LINKED ISSUE #408: Spotify and Youtube tabs will simultaneously play if one tab is playing and one is unpaused
### Prerequisites:
Mic permissions are enabled.
Have a playing Youtube tab and a paused Spotify tab and have the paused tab active.

### STR:
1. Use the shortcut or click on the mic icon from the browser toolbar in the Spotify tab.
2. Say or write an "Unpause" command.

### Expected result:
Spotify tab starts playing and the Youtube tab is paused.

### Actual result:
Both Spotify and Youtube tabs are playing.

### Notes:
Reproduced on Mac 10.14.6 and Win10x64 with Firefox Nightly 71.0a1 (64-bit).
Reproduced also for a paused Youtube tab and a playing Spotify tab, or for a paused Youtube tab and a playing Youtube tab.
I think it may be related to [346](https://github.com/mozilla/firefox-voice/issues/346).

### PR DESCRIPTION
**Pull request by shreyaa-s**
Before submitting a final PR, please:

- [x] Run `npm test` on your machine
- [x] Run `npm run format` to keep code formatted consistently

fixes #408 
The following change pauses every tab first and then unpauses the specified tab.

### CODE REVIEW

File: extension/intents/music/music.js [hunk 1]
```diff
@@ -94,6 +94,10 @@ intentRunner.registerIntent({
 intentRunner.registerIntent({
   name: "music.unpause",
   async run(context) {
+    for (const ServiceClass of Object.values(SERVICES)) {
+      const service = new ServiceClass(context);
+      await service.pauseAny();
```
Comment (ianb) [inline extension/intents/music/music.js]: Right here you could use `service.pauseAny({exceptTabId: (await browserUtils.activeTab()).id)` and then if the current tab is already playing it won't get paused and unpaused.

Comment (shreyaa-s) [reply extension/intents/music/music.js]: Requested changes have been made.

**Comment by shreyaa-s** [comment]
Requested changes have been made.

**Comment by ianb** [comment]
Sorry I lost track of this one for a few days. Looks good, thanks!

**Pull request merged**